src/index.spec.ts

```diff
@@ -104,173 +160,183 @@ describe('EppoJSClient E2E test', () => {
     mock.teardown();
   });
 
-  it('assigns subject from overrides when experiment is enabled', () => {
-    td.replace(EppoLocalStorage.prototype, 'get', (key: string) => {
-      if (key !== hashedFlagKey) {
-        throw new Error('Unexpected key ' + key);
-      }
-      return {
-        ...mockExperimentConfig,
-        overrides: {
-          '1b50f33aef8f681a13f623963da967ed': 'variant-2',
-        },
-        typedOverrides: {
-          '1b50f33aef8f681a13f623963da967ed': 'variant-2',
-        },
-      };
-    });
-
-    const assignment = globalClient.getAssignment('subject-10', flagKey);
-    expect(assignment).toEqual('variant-2');
-  });
-
-  it('assigns subject from overrides when experiment is not enabled', () => {
-    td.replace(EppoLocalStorage.prototype, 'get', (key: string) => {
-      if (key !== hashedFlagKey) {
-        throw new Error('Unexpected key ' + key);
-      }
-      return {
-        ...mockExperimentConfig,
-        overrides: {
-          '1b50f33aef8f681a13f623963da967ed': 'variant-2',
-        },
-        typedOverrides: {
-          '1b50f33aef8f681a13f623963da967ed': 'variant-2',
-        },
-      };
-    });
-    const assignment = globalClient.getAssignment('subject-10', flagKey);
-    expect(assignment).toEqual('variant-2');
-  });
-
-  it('returns null when experiment config is absent', () => {
+  it('returns default value when experiment config is absent', () => {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     td.replace(EppoLocalStorage.prototype, 'get', (key: string) => null as null);
-    const assignment = globalClient.getAssignment('subject-10', flagKey);
-    expect(assignment).toEqual(null);
+    const assignment = globalClient.getStringAssignment('subject-10', flagKey, '');
+    expect(assignment).toEqual('');
   });
 
   it('logs variation assignment and experiment key', () => {
     td.replace(EppoLocalStorage.prototype, 'get', (key: string) => {
-      if (key !== hashedFlagKey) {
+      if (key !== flagKey) {
```

Comment:
Do we ever expect the client-side JS client to ever _not_ use hashed/obfuscated configurations?
I could see ituseful for local debugging.

If so, I wonder if, in our top-level UFC object, we include `obfuscated=true/false` as a hint to the SDK if it's at play or not.